flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/agg/SortAggITCase.scala

```diff
@@ -364,6 +364,60 @@ class SortAggITCase extends AggregateITCaseBase("SortAggregate") {
       Seq(row(4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L, 4L))
     )
   }
+
+  @Test
+  def testJsonArrayAggAndJsonObjectAggWithOtherAggs(): Unit = {
+    val data =
+      List(rowOf(1, "A"), rowOf(3, "B"), rowOf(2, "C"))
```

Comment:
nit: can we reuse the wider test table below?

Reply:
> nit: can we reuse the wider test table below?

Done!